docs/reference/commandline/dockerd.md

```diff
@@ -302,6 +302,24 @@ options for `zfs` start with `zfs` and options for `btrfs` start with `btrfs`.
 
 #### Devicemapper options
 
+This is an example of the configuration file for devicemapper on Linux:
+
+```json
+{
+  "storage-driver": "devicemapper",
+  "storage-opts": [
+    "dm.thinpooldev=/dev/mapper/thin-pool",
+    "dm.use_deferred_deletion=true",
+    "dm.use_deferred_removal=true"
+  ],
+  "log-driver": "json-file",
```

Comment:
The line above this json block says this is an example for devicemapper options, seems weird to add log driver stuff here.